hack/build-index-image.sh

```diff
@@ -40,6 +40,11 @@ function create_index_image() {
 
   docker build -t "${BUNDLE_IMAGE_NAME}" -f bundle.Dockerfile --build-arg "VERSION=${CURRENT_VERSION}" .
   docker push "${BUNDLE_IMAGE_NAME}"
+
+  # Extract the digest of the bundle image, to be added to the index image
+  DIGEST=$(skopeo inspect docker://${BUNDLE_IMAGE_NAME} | jq .Digest | tr -d '"')
```

Comment:
A silly comment: first, the digester can do that, using the `--image` flag. But also, the `docker push` command also output the digest, so maybe it's possibel to capture it nin line 42?

Reply:
Thanks for the input.
I've changed it to use the digester tool. Prefer it over trying to capture the digest in `docker push` output.